tests/core/test_distributions.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import pytest
+import torch
+
+import scvi
+from scvi.metrics._core import unsupervised_clustering_accuracy
+from scvi.core._distributions import (
+    ZeroInflatedNegativeBinomial,
+    NegativeBinomial,
+)
+from scvi.core._log_likelihood import log_zinb_positive, log_nb_positive
+
+scvi.set_seed(0)
+use_cuda = True
+
+
+def test_deprecated_munkres():
+    y = np.array([0, 1, 0, 1, 0, 1, 1, 1])
+    y_pred = np.array([0, 0, 0, 0, 1, 1, 1, 1])
+    reward, assignment = unsupervised_clustering_accuracy(y, y_pred)
+    assert reward == 0.625
+    assert (assignment == np.array([[0, 0], [1, 1]])).all()
+
+    y = np.array([1, 1, 2, 2, 0, 0, 3, 3])
+    y_pred = np.array([1, 1, 2, 2, 3, 3, 0, 0])
+    reward, assignment = unsupervised_clustering_accuracy(y, y_pred)
+    assert reward == 1.0
+    assert (assignment == np.array([[0, 3], [1, 1], [2, 2], [3, 0]])).all()
+
+
+def test_zinb_distribution():
```

Comment:
In this test, I am not sure of what we are testing. We implemented both functions right? The log_nb and the classes. I understand at some point we wanted to make sure both return the same thing. Do we still want to test this thing?